src/views/static/about/about-view.js

```diff
@@ -0,0 +1,22 @@
+import { customElement, html, LitElement } from 'lit-element';
```

Comment:
Is this page being used at all?

Reply:
Same as above, it _should_ be use but it isn't atm.